docs/src/modules/components/demos/data-grid/Demos.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import PTOCalendar from './PTOCalendar/PTOCalendar';
+import ToggleButtonGroup, { toggleButtonGroupClasses } from '@mui/material/ToggleButtonGroup';
+import ToggleButton from '@mui/material/ToggleButton';
+import Paper from '@mui/material/Paper';
+import Stack from '@mui/material/Stack';
+import CalendarMonth from '@mui/icons-material/CalendarMonth';
+import BarChart from '@mui/icons-material/BarChart';
+import Box from '@mui/material/Box';
+import StockDashboard from './FinancialDashboard/StockDashboard';
+
+const DEMOS: { name: string; icon: React.ReactNode; component: React.ComponentType<any> }[] = [
```

Comment:
It would be great if we refrect the selected demo in the URL, so that tabs don't reset on page refresh